patches/minecraft/net/minecraft/client/gui/GuiScreen.java.patch

```diff
@@ -35,15 +35,20 @@
          {
              GlStateManager.func_179101_C();
              RenderHelper.func_74518_a();
-@@ -418,6 +428,7 @@
+@@ -414,10 +421,12 @@
+ 
+     public void func_175281_b(String p_175281_1_, boolean p_175281_2_)
+     {
++        if((p_175281_1_=net.minecraftforge.event.ForgeEventFactory.onClientSendMessage(p_175281_1_)).isEmpty())return;
```

Comment:
It is preferred NOT to use that type of coding. Builders are fine, because its a clear chain. 
It's not as clear when you're assigning a variable and using the synthetic variable after it.
Even with 20 years of java experience I still hate those style of comparisons because it's so easy to miss the assignment.